app/utils/command.py

```diff
@@ -0,0 +1,81 @@
+import inspect
+import logging
+from typing import Any
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandParser
+
+
+class CustomBaseCommand(BaseCommand):
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        """
+        The actual logic of the command. Subclasses must implement
+        this method.
+        """
+        raise NotImplementedError("subclasses of CustomBaseCommand must provide a handle() method")
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument('--logger', action='store_true', help='use logger configuration')
+
+
+class CommandHandler():
```

Comment:
Maybe add a comment to where this comes from (service-stac?) so that we know that changing stuff here should probably also go into service-stac

Reply:
Comment added.